### ISSUE TITLE
Improving QuickAccess/Searchbar 

### ISSUE THREAD

**Issue by tassadarius** [jcryptool/core#184]
Implementation issue for the record: #162

JCrypTool now has a Search-Icon using the eclipse functionality of `org.eclipse.ui.window.quickAccess`. It could be improved.

### Room for Improvement:

* Adjust the position of the search icon to be more prominent (e.g. focused on the right side as in eclipse)
* Try to optimize search results (filter unnecessary commands / allow tag search)

**Comment by simlei** [comment]
Cool :)
I'm excited to see it in action in the weekly build!

**Comment by simlei** [comment]
As of 2021, this works. _Concrete_ ideas for more improvements should be opened as separate feature requests.

**Comment by grthor** [comment]
I would suggest to reopen this issue because it is already a feature request for improvements of the quick access bar.

**Comment by simlei** [comment]
I'd be fine with that; question is, how feasible are the "room for improvement" options. As I recollect it, for neither we had seen a straightforward way of implementing. Therefore, I fear this issue may stay open forever + not contribute too much useful information, and my emphasis on _concrete_ ideas. Still want this reopened?

**Comment by grthor** [comment]
You're right. The quick access bar is a part of eclipse that is hard to modify.
Better let it closed, as we can participate on improvements done by eclipse by not modifying it.

**Comment by tassadarius** [comment]
The current hover tooltip is `Aktionen finden (Strg+3)` in German. We think that `quick search (ctrl + 3)` / `Schnellsuche (Strg + 3)` may be suited besser.
